src/main/java/org/mule/service/oauth/internal/AbstractOAuthDancer.java

```diff
@@ -141,8 +146,12 @@ public void start() throws MuleException {
 
   @Override
   public void stop() throws MuleException {
-    if (pollScheduler != null) {
-      pollScheduler.stop();
+    try {
+      if (pollScheduler != null) {
+        pollScheduler.stop();
+      }
+    } catch (Throwable t) {
+      LOGGER.warn("Throwable while stopping pollScheduler. Will continue stopping the dancer...", t);
```

Comment:
Make a nicer user friendly message. Found error trying to stop dancer ${name}. Execution will continue...